.github/workflows/tests.yml

```diff
@@ -42,6 +42,8 @@ jobs:
         NOSE_DIVIDED_WE_RUN: ${{ matrix.NOSE_DIVIDED_WE_RUN }}
         JS_SETUP: yes
       run: scripts/docker test --noinput --stop -v --divided-we-run=${{ matrix.NOSE_DIVIDED_WE_RUN }} --divide-depth=1 --with-timing --with-flaky --threshold=10 --max-test-time=29
+    - name: Dependency Tracking Info To Datadog
```

Comment:
Should we have a sepearte workflow for it? Or would it be okay a put in existing workflows?